packages/flutter/lib/src/cupertino/desktop_text_selection_toolbar.dart

```diff
@@ -10,23 +10,23 @@ import 'colors.dart';
 // the screen.
 const double _kToolbarScreenPadding = 8.0;
 
-// These values were measured from a screenshot of TextEdit on macOS 10.15.7 on
-// a Macbook Pro.
+// These values were measured from a screenshot of the native context menu on
+// macOS 13.2 on a Macbook Pro.
 const double _kToolbarWidth = 222.0;
-const Radius _kToolbarBorderRadius = Radius.circular(4.0);
-const EdgeInsets _kToolbarPadding = EdgeInsets.symmetric(
-  vertical: 3.0,
-);
+const Radius _kToolbarBorderRadius = Radius.circular(8.0);
+const EdgeInsets _kToolbarPadding = EdgeInsets.all(6.0);
 
-// These values were measured from a screenshot of TextEdit on macOS 10.16 on a
-// Macbook Pro.
-const CupertinoDynamicColor _kToolbarBorderColor = CupertinoDynamicColor.withBrightness(
-  color: Color(0xFFBBBBBB),
-  darkColor: Color(0xFF505152),
+// These values were measured from a screenshot of the native context menu on
+// macOS 13.2 on a Macbook Pro.
+const CupertinoDynamicColor _kToolbarBorderColor =
+    CupertinoDynamicColor.withBrightness(
+  color: Color(0xFFB8B8B8),
+  darkColor: Color(0xFF5B5B5B),
 );
-const CupertinoDynamicColor _kToolbarBackgroundColor = CupertinoDynamicColor.withBrightness(
-  color: Color(0xffECE8E6),
-  darkColor: Color(0xff302928),
+const CupertinoDynamicColor _kToolbarBackgroundColor =
+    CupertinoDynamicColor.withBrightness(
+  color: Color(0xFFECECEC),
+  darkColor: Color(0xFF333333),
 );
```

Comment:
Nit: I think this should be indented more like this, but not a big deal.

```dart
    CupertinoDynamicColor.withBrightness(
      color: Color(0xFFECECEC),
      darkColor: Color(0xFF333333),
    );
```

Reply:
@justinmc I used `dart format`, but I agree this is cleaner. Should I disregard the default formatting?